Week-4/callApplyBind/script.js

```diff
@@ -0,0 +1,64 @@
+// Calculator class with basic arithmetic methods
+class Calculator {
+    add(a, b) {
+      return a + b;
+    }
+  
+    subtract(a, b) {
+      return a - b;
+    }
+  
+    multiply(a, b) {
+      return a * b;
+    }
+  
+    divide(a, b) {
```

Comment:
Implement error handling for divide method when b is zero, even though you have already thrown an error. It's always good to handle such scenarios gracefully in a user interface.